packages/apps/hr/src/components/molecules/ExpenseDetailsValidationButton/ExpenseDetailsValidationButton.js

```diff
@@ -57,13 +67,28 @@ const ExpenseDetailsValidationButton = ({expense, mode}) => {
     );
   }, [dispatch, mode, expense]);
 
+  const deleteExpenseAPI = useCallback(() => {
+    dispatch(deleteExpense({ExpenseId: expense.id, userId: user.id}));
+    navigation.navigate('ExpenseListScreen');
```

Comment:
I think it's better to use `navigation.pop()` here because this screen with this expense id will no longer exist so it's better to remove it from the stack